lib/server.js

```diff
@@ -3,6 +3,7 @@
 var path = require('path');
 var fs = require('fs');
 var Promise = require('lie');
+var jsonParser = require('body-parser').json({limit: '1mb'});
```

Comment:
Is this only for parsing the auth response message? why so large? maybe doesn't matter, but curious why you had it so huge.